### PR TITLE
MANTA-4628 Rename extant buckets components such that the name reflects the function

### PR DESCRIPTION
**Pull request by kellymclaughlin**

### CODE REVIEW

File: lib/buckets/objects/metadata/update.js
Comment (trentm) [inline lib/buckets/objects/metadata/update.js]: nit: indentation

File: lib/buckets/common.js
Comment (trentm) [inline lib/buckets/common.js:25]: nit: there is a "boray" ref in a comment on line 70 here.

File: lib/buckets_mdapi_client.js
Comment (trentm) [inline lib/buckets_mdapi_client.js:71]: This was cruft?

File: package.json
Comment (trentm) [inline package.json]: You will update this before merging?

File: lib/buckets_mdapi_client.js
Comment (kellymclaughlin) [reply lib/buckets_mdapi_client.js:71]: Yeah, that was carryover from when this module lived in `electric-boray`/`buckets-mdplacement`.

File: package.json
Comment (kellymclaughlin) [reply package.json]: Yep, for sure

File: lib/buckets/common.js
Comment (kellymclaughlin) [reply lib/buckets/common.js:25]: Fixed

File: lib/buckets/objects/metadata/update.js
Comment (kellymclaughlin) [reply lib/buckets/objects/metadata/update.js]: Fixed

**Review by trentm — DISMISSED** [review]
LGTM, modulo the  package.json dep update.

**Review by bowrocker — APPROVED** [review]
LGTM

**Review by trentm — APPROVED** [review]

**Pull request merged**
